src/taipy/core/submission/submission.py

```diff
@@ -140,28 +141,40 @@ def _update_submission_status(self, plop: Job):
             if not job:
                 continue
             if job.is_failed():
-                submission_status = SubmissionStatus.FAILED
-                break
+                self.submission_status = SubmissionStatus.FAILED  # type: ignore
```

Comment:
We should type the argument in the signature of the submission_status property setter. So we don't need the type ignore.

Reply:
I believe I did but it still raises the attribute is read-only :/